src/code/views/app-view.coffee

```diff
@@ -0,0 +1,67 @@
+LinkView    = React.createFactory require './link-view'
+NodeWell    = React.createFactory require './node-well-view'
+NodeEditView= React.createFactory require './node-edit-view'
+LinkEditView= React.createFactory require './link-edit-view'
+StatusMenu  = React.createFactory require './status-menu-view'
```

Comment:
This is new to me.  If you have the time I would love to understand that 'createFactory' does in this case.